vpr/src/base/setup_vpr.cpp

```diff
@@ -24,29 +24,37 @@
 #include "clock_modeling.h"
 #include "ShowSetup.h"
 
-static void SetupNetlistOpts(const t_options& Options, t_netlist_opts& NetlistOpts);
-static void SetupAPOpts(const t_options& options,
-                        t_ap_opts& apOpts);
-static void SetupPackerOpts(const t_options& Options,
-                            t_packer_opts* PackerOpts);
-static void SetupPlacerOpts(const t_options& Options,
-                            t_placer_opts* PlacerOpts);
-static void SetupAnnealSched(const t_options& Options,
-                             t_annealing_sched* AnnealSched);
-static void SetupRouterOpts(const t_options& Options, t_router_opts* RouterOpts);
-static void SetupNocOpts(const t_options& Options,
-                         t_noc_opts* NocOpts);
-static void SetupServerOpts(const t_options& Options,
-                            t_server_opts* ServerOpts);
-
-static void SetupRoutingArch(const t_arch& Arch, t_det_routing_arch& RoutingArch);
-
-static void SetupTiming(const t_options& Options, const bool TimingEnabled, t_timing_inf* Timing);
-static void SetupSwitches(const t_arch& Arch,
-                          t_det_routing_arch& RoutingArch,
-                          const std::vector<t_arch_switch_inf>& arch_switches);
-static void SetupAnalysisOpts(const t_options& Options, t_analysis_opts& analysis_opts);
-static void SetupPowerOpts(const t_options& Options, t_power_opts* power_opts, t_arch* Arch);
+static void setup_netlist_opts(const t_options& Options, t_netlist_opts& NetlistOpts);
+static void setup_ap_opts(const t_options& options,
+                          t_ap_opts& apOpts);
+static void setup_packer_opts(const t_options& Options,
+                              t_packer_opts* PackerOpts);
+static void setup_placer_opts(const t_options& Options,
+                              t_placer_opts* PlacerOpts);
+static void setup_anneal_sched(const t_options& Options,
+                               t_annealing_sched* AnnealSched);
+static void setup_router_opts(const t_options& Options, t_router_opts* RouterOpts);
+static void setup_noc_opts(const t_options& Options,
+                           t_noc_opts* NocOpts);
+static void setup_server_opts(const t_options& Options,
+                              t_server_opts* ServerOpts);
+
+static void setup_routing_arch(const t_arch& Arch, t_det_routing_arch& RoutingArch);
+
+static void setup_timing(const t_options& Options, const bool TimingEnabled, t_timing_inf* Timing);
+static void setup_switches(const t_arch& Arch,
+                           t_det_routing_arch& RoutingArch,
+                           const std::vector<t_arch_switch_inf>& arch_switches);
+static void setup_analysis_opts(const t_options& Options, t_analysis_opts& analysis_opts);
+static void setup_power_opts(const t_options& Options, t_power_opts* power_opts, t_arch* Arch);
+
+static void setup_vib_inf(const std::vector<t_physical_tile_type>& PhysicalTileTypes,
```

Comment:
Should have a Doxygen comment saying what setup_vib_inf is and when it is used.  Need to say not needed unless ...